tools/quicktype-wrapper/src/index.ts

```diff
@@ -19,16 +19,16 @@ const OUT = argv.out || process.env.OUT;
 const L = (argv.l || process.env.L || LANGUAGE.TYPESCRIPT).toUpperCase() as TARGET_LANGUAGE;
 
 // The name of the JSON Schema file
-const SCHEMA_FILE = 'schema.json';
+const SCHEMA_PATTERN = '*.json';
 
 /**
  * Gets a list of all JSON Schema paths (absolute paths)
  * @param directory The path to the directory with schemas.
  */
 async function getJSONSchemasPaths(directory: string) {
-  console.log(`- Finding all JSON Schemas (${SCHEMA_FILE})...`);
+  console.log(`- Finding all JSON Schemas (${SCHEMA_PATTERN})...`);
   const recursive = require("recursive-readdir");
-  const paths: string[] = await recursive(directory, [`!${SCHEMA_FILE}`])
+  const paths: string[] = await recursive(directory, [`!${SCHEMA_PATTERN}`]);
```

Comment:
nit: This syntax was sufficiently obtuse I researched the library. Maybe a comment on excluding files without this pattern?

Reply:
Good point. I extracted this pattern and added a comment.